src/main/java/liquibase/ext/databricks/sqlgenerator/InsertOrUpdateGeneratorDatabricks.java

```diff
@@ -106,20 +103,17 @@ public Sql[] generateSql(InsertOrUpdateStatement insertOrUpdateStatement, Databa
         completeSql.append(updateStatement);
 
         // Add insert statement to MERGE if not ONLY an update statement
-        if (!insertOrUpdateStatement.getOnlyUpdate()) {
+        if (Boolean.FALSE.equals(insertOrUpdateStatement.getOnlyUpdate())) {
```

Comment:
Was it a complain because of autoboxing?
I have a bunch of similar checks with, but with primitive type.

Reply:
kinda, it complains about auto-Unboxing, saying in case of getOnlyUpdate() is `null` we'll get NPE